crates/oxc_minifier/tests/peephole/esbuild.rs

```diff
@@ -731,35 +731,35 @@ fn js_parser_test() {
     test("if ((a << +b) !== 0) throw 0", "if (a << +b) throw 0;");
     test("if ((a >> +b) !== 0) throw 0", "if (a >> +b) throw 0;");
     test("if ((a >>> b) !== 0) throw 0", "if (a >>> b) throw 0;");
-    // test("if (+a !== 0) throw 0", "if (+a != 0) throw 0;");
-    // test("if (~a !== 0) throw 0", "if (~a) throw 0;");
+    test("if (+a !== 0) throw 0", "if (+a != 0) throw 0;");
+    test("if (~a !== 0) throw 0", "if (~a !== 0) throw 0;");
```

Comment:
See https://github.com/evanw/esbuild/issues/4065 for the reason for these changes.